workshop/4-auth-me-baby.md

```diff
@@ -140,17 +144,17 @@ To use a dropdown in bootstrap we need the bootstrap javascript. To keep things
 Open `/app/index.html`, and below the script tag that brings in `ambitious-arcgis-app.js`, add this license
 
 ```
-<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" crossorigin="anonymous"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
 ```
 
-Now, open up `/app/application/template.hbs` and replace the code between `{{#if session.isAuthenticated}}` and `{{else}}` as shown below.
+Now, open up `/app/templates/application.hbs` and replace the code between `{{#if session.isAuthenticated}}` and `{{else}}` as shown below.
 
 ```
 {{#if session.isAuthenticated}}
-<li class="dropdown">
-  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">{{session.currentUser.fullName}} <span class="caret"></span></a>
+<li class="nav-item dropdown">
```

Comment:
I think you'll need to update this to something like: https://github.com/mjuniper/ambitious-arcgis-app-2018/pull/8#issuecomment-369443772